doc/GeneratorMigration/Library_Inventory.md

```diff
@@ -0,0 +1,484 @@
+| servicefabricmanagedclusters | Azure.ResourceManager.ServiceFabricManagedClusters | sdk/servicefabricmanagedclusters/Azure.ResourceManager.ServiceFabricManagedClusters |
+| servicelinker | Azure.ResourceManager.ServiceLinker | sdk/servicelinker/Azure.ResourceManager.ServiceLinker |
+| signalr | Azure.ResourceManager.SignalR | sdk/signalr/Azure.ResourceManager.SignalR |
+| sphere | Azure.ResourceManager.Sphere | sdk/sphere/Azure.ResourceManager.Sphere |
+| springappdiscovery | Azure.ResourceManager.SpringAppDiscovery | sdk/springappdiscovery/Azure.ResourceManager.SpringAppDiscovery |
+| sqlmanagement | Azure.ResourceManager.Sql | sdk/sqlmanagement/Azure.ResourceManager.Sql |
+| sqlvirtualmachine | Azure.ResourceManager.SqlVirtualMachine | sdk/sqlvirtualmachine/Azure.ResourceManager.SqlVirtualMachine |
+| storage | Azure.ResourceManager.Storage | sdk/storage/Azure.ResourceManager.Storage |
+| storageactions | Azure.ResourceManager.StorageActions | sdk/storageactions/Azure.ResourceManager.StorageActions |
+| storagecache | Azure.ResourceManager.StorageCache | sdk/storagecache/Azure.ResourceManager.StorageCache |
+| storagemover | Azure.ResourceManager.StorageMover | sdk/storagemover/Azure.ResourceManager.StorageMover |
+| storagepool | Azure.ResourceManager.StoragePool | sdk/storagepool/Azure.ResourceManager.StoragePool |
+| storagesync | Azure.ResourceManager.StorageSync | sdk/storagesync/Azure.ResourceManager.StorageSync |
+| streamanalytics | Azure.ResourceManager.StreamAnalytics | sdk/streamanalytics/Azure.ResourceManager.StreamAnalytics |
+| subscription | Azure.ResourceManager.Subscription | sdk/subscription/Azure.ResourceManager.Subscription |
+| support | Azure.ResourceManager.Support | sdk/support/Azure.ResourceManager.Support |
+| synapse | Azure.ResourceManager.Synapse | sdk/synapse/Azure.ResourceManager.Synapse |
+| trafficmanager | Azure.ResourceManager.TrafficManager | sdk/trafficmanager/Azure.ResourceManager.TrafficManager |
+| trustedsigning | Azure.ResourceManager.TrustedSigning | sdk/trustedsigning/Azure.ResourceManager.TrustedSigning |
+| voiceservices | Azure.ResourceManager.VoiceServices | sdk/voiceservices/Azure.ResourceManager.VoiceServices |
+| webpubsub | Azure.ResourceManager.WebPubSub | sdk/webpubsub/Azure.ResourceManager.WebPubSub |
+| websites | Azure.ResourceManager.AppService | sdk/websites/Azure.ResourceManager.AppService |
+| workloadmonitor | Azure.ResourceManager.WorkloadMonitor | sdk/workloadmonitor/Azure.ResourceManager.WorkloadMonitor |
+| workloads | Azure.ResourceManager.Workloads | sdk/workloads/Azure.ResourceManager.Workloads |
+
+
+## Libraries with Unknown Generator
+
+| Service | Library | Path |
+| ------- | ------- | ---- |
+| batch | Microsoft.Azure.Batch.Conventions.Files | sdk/batch/Microsoft.Azure.Batch.Conventions.Files |
+| batch | Microsoft.Azure.Batch.FileStaging | sdk/batch/Microsoft.Azure.Batch.FileStaging |
+| cloudmachine | Azure.Projects | sdk/cloudmachine/Azure.Projects |
```

Comment:
If there is no autorest.md or tsp-location.yaml it should treat this as No Generator not Unknown Generator